### ISSUE TITLE
[Request] Allow hiding the side menu when in desktop mode and pane width is small.

### ISSUE THREAD

**Issue by sytone** [Quorafind/Obsidian-Memos#127]
At the moment there is a trigger on resize for the window. That can be added to the resize of the leaf so when that happens allow the side bar to hide as well. This allows for a  view of only the memos without the maps and tag list when you have it open as a pane but small. 

Looks like CSS would need to be updated as it has flags for screen and width in it. 

The view would look something like this with the side bar hidden when the pane with is small.

![image](https://user-images.githubusercontent.com/1399443/154354876-5eddf871-3a54-4054-b595-62aef25c7991.png)

**pseudocode**

in onload()

```
        this.registerEvent(
            this.app.workspace.on("resize", () => {
                this.handleResize();
            })
        );
```

Function in MemosPlugin

```
    async handleResize() {
        console.log("handleResize");
        const leaves = this.app.workspace.getLeavesOfType(MEMOS_VIEW_TYPE);
        if (leaves.length > 0) {
            const leaf = leaves[0];
            console.log(leaf.width);
            if(leaf.width <= 875) {
              // hide the sidebar
            }

        }
    }
```

**Comment by Quorafind** [comment]
Thanks for tips, will add this feature soon

**Comment by Quorafind** [comment]
Released in 1.9.0
